src/app/modules/pages/door-permissions/door-permissions.component.ts

```diff
@@ -1,19 +1,45 @@
-import { Component, OnInit } from '@angular/core';
-import { NgxSpinnerService } from 'ngx-spinner';
+import {Component, OnInit} from '@angular/core';
+import {NgxSpinnerService} from 'ngx-spinner';
+import {Observable} from 'rxjs';
+import {DoorPermissionModel} from '../../../shared/models/door-permission.model';
+import {DoorHttpService} from '../../../core/http-services/door.http-service';
+import {Router} from '@angular/router';
 
 @Component({
-  selector: 'app-door-permissions',
-  templateUrl: './door-permissions.component.html'
+    selector: 'app-door-permissions',
```

Comment:
This is a page, so selector is not needed. You dont use it inside another component as <app-door-permissions></app-door-permissions> then selector isnt needed.